benchmarks/styled-components/package.json

```diff
@@ -1,13 +1,15 @@
 {
   "private": true,
   "scripts": {
+    "benchmark": "node ./system.js",
     "start": "webpack-dev-server"
   },
   "devDependencies": {
     "@babel/core": "^7.4.5",
     "@babel/preset-env": "^7.4.5",
     "@babel/preset-react": "^7.0.0",
     "babel-loader": "^8.0.6",
+    "benchmark": "^2.1.4",
```

Comment:
I think it is not part of this one.

Reply:
The way it is right now, the file `system.js` (which runs the CLI benchmark against `styled-system`) is outside the folder with the `package.json`. If I try to run `node system.js` I'll simple get the error `Error: Cannot find module 'benchmark'` because I don't have it installed.

The two possible solutions are:

1) Install `benchmark` globally with `npm i -g benchmark`. This is a terrible solution because you're forcing someone to install a library globally to run a local test. I don't want to pollute my machine with libraries I won't use again.

2) Move `system.js` inside the benchmark folder, with the `package.json`, and install `benchmark` as a dependency so people don't need to install it globally. I also included a npm script to run this file inside `package.json`. As you said before, the benchmarks are not all related to `styled-components`, so a single project for all the benchmarks should be used.

Please note this isn't the library `package.json`, it's merely the one inside the benchmark folder.